packages/hint-create-element-svg/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "Infrom users that they need to use createElementNS to create SVG elements instead of createElement",
+  "devDependencies": {
+    "@hint/parser-javascript": "^2.0.3",
+    "@hint/utils-tests-helpers": "^2.0.3",
+    "@types/debug": "0.0.31",
+    "@types/node": "10.12.18",
+    "ava": "^0.25.0",
+    "cpx": "^1.5.0",
+    "eslint": "^5.12.0",
+    "eslint-plugin-markdown": "^1.0.0",
+    "eslint-plugin-typescript": "0.14.0",
```

Comment:
```suggestion
    "@typescript-eslint/eslint-plugin": "^1.3.0",
```

Reply:
@sarvaje @molant Do we still need to fix this in the `create-hint` package?

Reply:
No, I just need to publish a new version and I've been a bit busy this week 😔